presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -1242,14 +1230,26 @@ private void compareResults(MaterializedResult actual, MaterializedResult expect
     {
         // Verify result count
         assertEquals(actual.getRowCount(),
-                expected.getRowCount(), String.format("Expected %d rows got %d rows", expected.getRowCount(), actual.getRowCount()));
+                expected.getRowCount(), format("Expected %d rows got %d rows", expected.getRowCount(), actual.getRowCount()));
         if (checkOrdering) {
             assertEquals(actual.getMaterializedRows(), expected.getMaterializedRows(), "Correctness check failed! Rows are not equal");
             return;
         }
         QueryAssertions.assertEqualsIgnoreOrder(actual, expected, "Correctness check failed! Rows are not equal");
     }
 
+    private void checkCTEInfoMatch(String explain, List<CTEInformation> expectedCTEInfoValues)
+    {
+        Matcher matcher = CTE_INFO_MATCHER.matcher(explain);
+        assertTrue(matcher.find());
+
+        String cteInfo = matcher.group();
+        for (CTEInformation value : expectedCTEInfoValues) {
+            assertTrue(cteInfo.contains(value.getCteName() + ": " + value.getNumberOfReferences() + " (is_view: " + value.getIsView() + ")"),
```

Comment:
Need to check the is materialized

Reply:
The Explain plan doesn't contain information if the CTE is materialized. It is of the form "CTEInfo: [temp: 2 (is_view: false)]". Is there some other way to check?

Reply:
Seems like a bug, let me see if we can fix this

Reply:
https://github.com/prestodb/presto/pull/23288

Reply:
That was quick. I wait for your changes in Issue #23288 to be merged and then add the same materialized check in these tests.

Reply:
@jaystarshot - I noticed they merge your commit today. I will update the tests to take into account isMaterialized.

Reply:
Updated the verifyResults method to check that the expected is_materialized value appears in the verbose Explain plan.